requirements_lock_3_10.txt

```diff
@@ -0,0 +1,153 @@
+#
+# This file is autogenerated by pip-compile with Python 3.10
```

Comment:
in the future how do we update/generate these requirements lock files?

Reply:
`bazel run //:requirements.update` to update requirements matching your system python version.
`bazel run //:requirements.update --repo_env=HERMETIC_PYTHON_VERSION=3.8` to update for a specific version.

The JAX's https://jax.readthedocs.io/en/latest/developer.html#managing-hermetic-python describes everythign in detail.

Once this is pushed I'll make a xla-specific, shortened version of that and update your documentation as well.